parser/src/python.lalrpop

```diff
@@ -69,6 +69,7 @@ PassStatement: ast::Statement = {
     <location:@L> "pass" => {
         ast::Statement {
             location,
+            custom: Default::default(),
```

Comment:
Is there a reason these can't be `custom: ()` as well? If you were looking to eventually have the parser be generic, `custom: ()` can be easily find/replaced eventually.

Reply:
I was looking to have the parser be generic later, but `()` seems to be clearer. will fix.